src/libcore/char/methods.rs

```diff
@@ -661,7 +661,7 @@ impl char {
     /// Returns `true` if this `char` is alphanumeric.
     ///
     /// 'Alphanumeric'-ness is defined in terms of the Unicode General Categories
-    /// 'Nd', 'Nl', 'No' and the Derived Core Property 'Alphabetic'.
+    /// 'Nd', 'Nl', 'No' and the Derived Core Property `Alphabetic`.
```

Comment:
I disagree with this change; why is this category in backticks but the others are not?

Reply:
I followed the convention used for documenting a few other methods (`is_lowercase`, `is_uppercase`, `is_whitespace`), where all `Unicode Derived Core Properties` are marked this way. After a second glance though, I think it would make more sense, if all properties using single quotes, would be formatted the same way. I'm also talking about the docs for many other methods in this section.

Tell me, what do you think, and I'll make the necessary changes.

Reply:
Yes, I think it makes sense for all Unicode categories/properties to be consistent. I think changing them to be inline code via backticks is reasonable <sup>1</sup>.

---

1 — although I don't think it's actually the "best" solution as they aren't really code. Perhaps it would have been better if they were all _emphasized_ or **bolded**. That's probably a change too far for now though.

Reply:
I'm on it.

Reply:
Done: e94e358